src/components/document/document-content.tsx

```diff
@@ -242,21 +241,22 @@ export class DocumentContentComponent extends BaseComponent<IProps, IState> {
           // below the last row - highlight bottom of last row
           ((i === rowElements.length - 1) && (e.clientY > rowBounds.bottom))) {
         dropInfo.rowDropIndex = i;
-        dropInfo.dropOffsetLeft = Math.abs(e.clientX - rowBounds.left);
-        dropInfo.dropOffsetTop = Math.abs(e.clientY - rowBounds.top);
-        dropInfo.dropOffsetRight = Math.abs(rowBounds.right - e.clientX);
-        dropInfo.dropOffsetBottom = Math.abs(rowBounds.bottom - e.clientY);
+
+        const dropOffsetLeft = Math.abs(e.clientX - rowBounds.left);
+        const dropOffsetTop = Math.abs(e.clientY - rowBounds.top);
+        const dropOffsetRight = Math.abs(rowBounds.right - e.clientX);
+        const dropOffsetBottom = Math.abs(rowBounds.bottom - e.clientY);
 
         const kSideDropThreshold = rowBounds.width * 0.25;
-        if ((dropInfo.dropOffsetLeft < kSideDropThreshold) &&
-            (dropInfo.dropOffsetLeft < dropInfo.dropOffsetRight!)) {
+        if ((dropOffsetLeft < kSideDropThreshold) &&
+            (dropOffsetLeft < dropOffsetRight!)) {
```

Comment:
It seems like you shouldn't need the `!` at this point.

Reply:
:+1: